config/OptionsBlocklist.lua

```diff
@@ -222,9 +222,81 @@ local function CreateNPCBlocklistOptions()
 	end)
 end
 
+local function CreateTitleBlocklistOptions()
+	local panel = CreateOptionsPanel('TitleBlocklist',
+		L['Title Blocklist'],
+		L['Quests with titles that partially match or IDs that exactly match entries from this list will not be automated.'],
+		L['Block Title'])
+
+	local function OnRemove(self)
+		for i, title in ipairs(ns.db.profile.blocklist.title) do
+			if title == self.title then
+				tremove(ns.db.profile.blocklist.title, i)
```

Comment:
Use `ns.db.profile.blocklist.title[i] = nil`.

Reply:
This would leave a gap in the array that would need to be guarded against elsewhere.